code/datums/status_effects/neutral.dm

```diff
@@ -135,3 +135,126 @@
 	name = "Holding Up"
 	desc = "You're currently pointing a gun at someone."
 	icon_state = "aimed"
+
+/datum/status_effect/aegis
+	id = "protected"
+	duration = 10 SECONDS
+	examine_text = "<span class='notice'>They are being guarded closely.</span>"
+	alert_type = /obj/screen/alert/status_effect/protected
+	///We need both the protector and protectee for this status effect, so let's just handle the protector's alert here too
+	var/alert_type_protector = /obj/screen/alert/status_effect/protector
+	///As above
+	var/obj/screen/alert/status_effect/linked_alert_protector
+	///The person tanking the damage
+	var/mob/living/protector
+	///We make the protectee partially transparent, so we need to remember if they were already partially transparent
+	var/original_alpha = 255
+	///All incoming brute/burn/stamina damage on the protector is multiplied by this, whether it was directed at the protectee or not
+	var/defensive_buff_factor = 0.75
+	///Paralyze the protectee for this long so they don't accidentally slip away prematurely
```

Comment:
return of the stungloves what it is what it ain't what it isn't

Reply:
well it's a good thing you get a big alert that tells you

Reply:
yes, because you're gonna take the time to hover your mouse over and read the text of an alert in the middle of combat.